### PR TITLE
Videomaker trial: Skip the launchpad

### PR DESCRIPTION
**Pull request by roundhill**
This PR removes the launchpad for the trial version of the Videomaker flow.

## Testing Instructions

* You must test on local dev
* Visit `/setup/videopress` and set up a site.
* After choosing the domain, you should be taken right to the site editor.
* Navigating to `wp-admin` shouldn't redirect you to the launchpad.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/videopress.ts [hunk 1]
```diff
@@ -169,19 +169,21 @@ const videopress: Flow = {
...
+						blogdescription: siteDescription,
+					} );
+					clearOnboardingSiteOptions();
+					return window.location.assign( `/site-editor/${ newSite.site_slug }` );
+				}
+
 				saveSiteSettings( newSite.blogid, {
 					launchpad_screen: 'full',
 					blogdescription: siteDescription,
 				} );
```
Comment (thedebian) [inline client/landing/stepper/declarative-flow/videopress.ts:185]:
```suggestion
				const isVideomakerTrial = config.isEnabled( 'videomaker-trial' );

				saveSiteSettings( newSite.blogid, {
					launchpad_screen: isVideomakerTrial ? 'off' : 'full',
					blogdescription: siteDescription,
				} );

				if ( isVideomakerTrial ) {
					clearOnboardingSiteOptions();
					return window.location.assign( `/site-editor/${ newSite.site_slug }` );
				}
```

Comment (thedebian) [reply client/landing/stepper/declarative-flow/videopress.ts:185]: It's fine as-is, but we could also remove the duplicated saveSiteSettings call if needed

**Review by thedebian — APPROVED** [review]
Working great, no more launchpad 🚀

File: client/landing/stepper/declarative-flow/videopress.ts (hunk 1 above)
Comment (roundhill) [reply client/landing/stepper/declarative-flow/videopress.ts:185]: I thought about doing it that way, but figured it was maybe easier to grok if there's only one block that handles all the videomaker trial stuff?

**Pull request merged**